asset/sass/blocks/_anchorjs.scss

```diff
@@ -0,0 +1,5 @@
+.anchorjs-link {
+    &:hover {
+        text-decoration : none;
+    }
+}
```

Comment:
Please add empty line at the end of the file

Reply:
Yes, the empty line is missing, but see my comment above to the topic of formatting.

Reply:
Ok, missed that, sorry. Agree but I think anyway we should have it there. This is global standard, and as you can see github is pointing it as well. Definitely code formatting here is not important, but it would be nice to keep some minimal standard, to not finish up with code like:
```css
.my-clAASss{ color: red; background-color:#812;   text-align: right !important;
float  :   right;}
```

:smile: